docs/notes-pa.md

```diff
@@ -3,4 +3,166 @@ how the permutation argument is formed.
 
 
 Permutation argument
-==============
\ No newline at end of file
+==============
+
+Within PLONK, the permutation 
+polynomials are used to compare 
+arguments about the proving 
+scheme.  
+
+These arguments are 
+checking that wire values, 
+within the circuit, are
+being correctly copied. 
```

Comment:
Thanks for resolving the previous issue. 

Maybe you could remove all mention of circuits/wire values and SNARK schemes to explain how the permutation argument works, then at the end, you explain how the permutation argument pertains to SNARK schemes and PLONK?

At the moment, it sounds a bit vague and I'm not sure I understand the use of the word "argument" in this context.

Reply:
Okay, I can pull the description away from wires.

Reply:
Then add the changes for PLONK at the end

Reply:
I've removed the mention of wires in the example, and written what the columns represent.